test/tests/SVGPathTest.coffee

```diff
@@ -31,6 +33,119 @@ describe "SVGPath", ->
 			point.x.should.be.closeTo 0, 0.01
 			point.y.should.be.closeTo 50, 0.01
 
+	describe "strokeLength", ->
```

Comment:
longer than length? less than zero, etc? Test those too?

Reply:
Correct, there weren't any tests, but the code was correct. I've added them for completeness